src/ActionModal/components/ActionModal.vue

```diff
@@ -106,8 +106,12 @@
           />
         </div>
         <div v-else-if="step === signStep" class="action-modal-form">
+          <!-- fallback to prevent the user to submit a tx before the sequence was loaded -->
+          <TmDataLoading
+            v-if="!context.account || context.account.sequence === undefined"
```

Comment:
```suggestion
            v-if="!context.account || !Number.isInteger(context.account.sequence)"
```

Reply:
showing a loading screen doesn't help and won't go away if it is undefined / null so i'm not sure i get the need for this.

Reply:
nicer then letting the user submit and then it crashes?

Reply:
i may be misunderstanding this, but this doesn't make sense to me.